internal/auth0/connection/expand.go

```diff
@@ -180,6 +181,141 @@ func expandConnectionOptionsGitHub(data *schema.ResourceData, config cty.Value)
 	return options, diag.FromErr(err)
 }
 
+func expandConnectionOptionsAttributes(config cty.Value) *management.ConnectionOptionsAttributes {
+	log.Printf("config: %v", config)
```

Comment:
Does this logging do anything useful? I don't think it even shows up when running terraform